authentification/transport/http/server.go

```diff
@@ -0,0 +1,16 @@
+package http
+
+import (
+	"os"
+
+	"github.com/There-is-Go-alternative/GoMicroServices/authentification/usecase"
+	"github.com/gofiber/fiber"
+)
+
+func NewHttpServer(u usecase.LoginProto) {
+	app := fiber.New()
+	//httpHandler := http.Handler{Logger: log.New(os.Stdout).With().Logger()}
+	app.Post("/login", LoginHandler(usecase.Login()))
+
+	app.Listen(os.Getenv("PORT"))
+}
```

Comment:
Listen est bloquant non ?
Du coup si tu veux faire autre chose dans ton main t'es niquée
La solution la plus simple et élégante pour gérer ça, c'est une go routine qui va lancer ton serveur.
```suggestion
import netHTTP "net/http"

func NewHttpServer(u usecase.LoginProto) *fiber.App {
	app := fiber.New()
	app.Post("/login", LoginHandler(usecase.Login()))
	
	return app
}
```
et dans ton main:
```
func main() {
	server := http.NewHttpServer(usecase.Login())
	go func() {
	     errc <- server.Listen(os.GetEnv("PORT"))
	}
	err := <- errc
	if err != nil {
	     log.Fatal(err)
	}
}
```
ou même mieux en te calant sur les interfaces http (si app.Handler() renvoie bien un http.Handler

```suggestion
import netHTTP "net/http"

func NewHttpServer(u usecase.LoginProto, port string) http.Handler {
	app := fiber.New()
	app.Post("/login", LoginHandler(usecase.Login()))
	
	return app,Handler()
}
```
et dans ton main:
```
func main() {
	server := http.NewHttpServer(usecase.Login())
	go func() {
	     errc <- ListenAndServe(os.GetEnv("PORT"), handler
	}
	err := <- errc
	if err != nil {
	     log.Fatal(err)
	}
}
```
https://docs.gofiber.io/api/app#handler
https://pkg.go.dev/net/http#Handler

https://pkg.go.dev/net/http#ListenAndServe


ou encore: 
https://github.com/There-is-Go-alternative/GoMicroServices/blob/dev/account/infra/http/gin.go
et lancer de la même manière ta méthode Run dans une go routine